app/models/messages/Consignment.scala

```diff
@@ -25,13 +25,14 @@ case class Consignment(
   inlandModeOfTransport: Option[String],
   TransportEquipment: Option[List[TransportEquipment]],
   LocationOfGoods: Option[LocationOfGoods],
-  DepartureTransportMeans: Option[DepartureTransportMeans],
+  DepartureTransportMeans: Option[List[DepartureTransportMeans]],
```

Comment:
There looks to be a lot of changes related to this. Could it be worth exploring changing this back to `Option[DepartureTransportMeans]` and defining the reads to read it in as a list of DepartureTransportMeans and take the head option?